requirements/base.txt

```diff
@@ -15,6 +15,8 @@ asgiref==3.6.0
     #   django-cors-headers
 asn1crypto==1.5.1
     # via webauthn
+async-timeout==4.0.3
```

Comment:
Why do we have a new dependency? We haven't updated redis version here. right?

Reply:
This was caused by generating the requirements with python 3.10, see https://github.com/maykinmedia/objecttypes-api/pull/129/commits/aeac44ac25418156e243a2c03d6d526a4bb20774.